content/api/cli.md

```diff
@@ -68,37 +76,46 @@ webpack index=./src/index.js entry2=./src/index2.js dist/bundle.js
 **List all of the options available on the cli**
 
 ```bash
-webpack --help , webpack -h
+webpack --help
+webpack -h
 ```
 
 **Build source using a config file**
 
-Specifies a different configuration file to pick up. Use this if you want to specify something different than `webpack.config.js`, which is the default.
+Specifies a different [configuration](/configuration) file to pick up. Use this if you want to specify something different than `webpack.config.js`, which is the default.
 
 ```bash
 webpack --config example.config.js
 ```
 
-**Send environment variable to be used in webpack config file**
+**Specify build environment**
+
+Send [environment](/configuration/environment) variable to be used in webpack config file.
+
 
 ```bash
 webpack --env=DEVELOPMENT
```

Comment:
It would be good to show the object variant here and how that works. There are some gotchas.